src/index.js

```diff
@@ -308,34 +307,20 @@ const announcement = new Announcement();
             console.log('DEBUG MODE ON!');
         }
 
-        // Get current site moderators
-        // Have to use /users/moderators instead of /users/moderators/elected because we also want appointed mods
-        const modURL = new URL(`${apiBase}/${apiVer}/users/moderators`);
-        modURL.search = new URLSearchParams({
-            pagesize: "100",
-            order: "desc",
-            sort: "reputation",
-            site: electionSiteApiSlug,
-            filter: "!LnNkvq0d-S*rS_0sMTDFRm",
-            key: stackApikey
-        }).toString();
-
         // Get current site named badges
-        if(!isStackOverflow) {
-
+        if (!isStackOverflow) {
             const allNamedBadges = await getAllNamedBadges(BotConfig, electionSiteApiSlug, stackApikey);
-            electionBadges.forEach(electionBadge => {
-                const matchedBadge = allNamedBadges.filter(namedBadge => electionBadge.name === namedBadge.name);
```

Comment:
This was just a quick fix, need to see if there are non-English badge names for the non-English sites. If yes we will probably need to hard-code them in a flat file.